docs/020-deployment/howtos/080-configure-dns.md

```diff
@@ -81,8 +81,15 @@ If you have any questions about DNS configuration, you can request CluedIn suppo
     global:
       dns:
         hostname: 1.2.3.4.sslip.io
+        # subdomains:
+          # openrefine: cluedin-clean-dev
+          # application: cluedin-app-dev
+        
+        # If you want to deviate away from the standard 'app' and 'clean' subdomains, you need to add the
+        # `subdomains` block of code and ensure that they match in the global.ingress.tls.hosts section as well.
     ```
-1. Edit the value of the host name to reflect your main domain. If you have 3 environments (development, test, and production), update the value of the host name for each environment (`dev.mycompany.com`, `test.mycompany.com`, `prod.mycompany.com.`).
+    **Note**: For environments sharing a SLD such as `customer.com`, it's important to plan the domains in advance to avoid issues during the onboarding process. 
```

Comment:
```suggestion
    **Note**: For environments sharing a second-level domain such as `customer.com`, it's important to plan the domains in advance to avoid issues during the onboarding process. 
```